Doc/library/warnings.rst

```diff
@@ -588,6 +588,9 @@ Available Context Managers
     Each object in the list has attributes with the same names as the arguments to
     :func:`showwarning`.
 
+    The definition of *category* and *lineno* are same as defined
+    at the top of :ref:`warning-filter` field.
+
```

Comment:
I would put it after the documentation on the *action* argument.

```rst
	See :ref:`warning-filter` for the meaning of the *category* and *lineno*
	parameters.
```

Reply:
Understood~